hydra-node/src/Hydra/Chain/Direct/Handlers.hs

```diff
@@ -189,12 +165,8 @@ finalizeTx TinyWallet{sign, getUTxO, coverFee} headState partialTx = do
     Right validatedTx -> do
       pure $ sign validatedTx
 
```

Comment:
Note that, while we're at it, this `CannotCoverFees` error is quite misleading here because it actually wraps several cases, some of which have nothing to do with covering fees so-to-speak:

```hs
  = ErrNoAvailableUTxO
  | ErrNotEnoughFunds ChangeError
  | ErrNoPaymentUTxOFound
  | ErrScriptExecutionFailed (RdmrPtr, ScriptFailure StandardCrypto)
```